useful workflows/end_at_timestamp.wdl

```diff
@@ -0,0 +1,77 @@
+version 1.0
+
+workflow end_at_timestamp {
+  input {
+    Int final_timestamp
```

Comment:
I think it might be a good idea to add a comment here stating the timestamp is in seconds?